### DISCUSSION TITLE
Activity Graph Not Working

### DISCUSSION THREAD

**Discussion by Coollaitar** [Ashutosh00710/github-readme-activity-graph#162] [Q&A]
https://github.com/Coollaitar/Coollaitar/blob/main/README.md
Please Help!

**Comment by CBID2** [comment]
> https://github.com/Coollaitar/Coollaitar/blob/main/README.md
> Please Help!

I’m going through this problem too @Coollaitar. Hopefully @Ashutosh00710 will respond promptly.
[https://github.com/CBID2](url)

**Reply by Ashutosh00710** [reply]
@Coollaitar @CBID2, It's working now

**Reply by CBID2** [reply]
Thanks @Ashutosh00710! Was there a bug?

**Reply by Ashutosh00710** [reply]
Nope, refer [this](https://github.com/Ashutosh00710/github-readme-activity-graph/issues/55)
